### DISCUSSION TITLE
Is possible to use it without system-priv ?

### DISCUSSION THREAD

**Discussion by eakteam** [Mygod/VPNHotspot#447] [Q&A]
Will this works if the app will have root access but not installed at system provileged apps?

**Comment by Mygod** [comment]
You do not need systemizing for almost all features in the app. See https://github.com/Mygod/VPNHotspot#features-that-requires-system-app-installation.

**Comment by nibazshab** [comment]
Yes, just installed it as usual, and give it ROOT, it will work
